xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/v2/KucoinV2ExceptionClassifier.java

```diff
@@ -0,0 +1,39 @@
+package org.knowm.xchange.kucoin.v2;
+
+import org.knowm.xchange.exceptions.ExchangeSecurityException;
+import org.knowm.xchange.exceptions.NonceException;
+
+import com.kucoin.sdk.exception.KucoinApiException;
+
+public final class KucoinV2ExceptionClassifier {
+
+  KucoinV2ExceptionClassifier() {}
+
+  public static RuntimeException classify(KucoinApiException e) {
+    if (e.getMessage().contains("check environment variables")) {
+      return new ExchangeSecurityException(e.getMessage(), e);
+    }
+
+    // TODO need to handle HTTP error codes which aren't covered
```

Comment:
See https://github.com/knowm/XChange/issues/2914#issuecomment-464319161 for query on this one.